lib/internal/freeze_intrinsics.js

```diff
@@ -129,34 +171,35 @@ module.exports = function() {
   const intrinsics = [
     // Anonymous Intrinsics
     // ThrowTypeError
-    getOwnPropertyDescriptor(Function.prototype, 'caller').get,
+    ObjectGetOwnPropertyDescriptor(Function.prototype, 'caller').get,
     // IteratorPrototype
-    getPrototypeOf(
-      getPrototypeOf(new Array()[Symbol.iterator]())
+    ObjectGetPrototypeOf(
+      ObjectGetPrototypeOf(new Array()[SymbolIterator]())
     ),
     // ArrayIteratorPrototype
-    getPrototypeOf(new Array()[Symbol.iterator]()),
+    ObjectGetPrototypeOf(new Array()[SymbolIterator]()),
     // StringIteratorPrototype
-    getPrototypeOf(new String()[Symbol.iterator]()),
+    ObjectGetPrototypeOf(new String()[SymbolIterator]()),
     // MapIteratorPrototype
-    getPrototypeOf(new Map()[Symbol.iterator]()),
+    ObjectGetPrototypeOf(new Map()[SymbolIterator]()),
     // SetIteratorPrototype
-    getPrototypeOf(new Set()[Symbol.iterator]()),
+    ObjectGetPrototypeOf(new Set()[SymbolIterator]()),
     // GeneratorFunction
-    getPrototypeOf(function* () {}),
+    ObjectGetPrototypeOf(function* () {}),
     // AsyncFunction
-    getPrototypeOf(async function() {}),
+    ObjectGetPrototypeOf(async function() {}),
     // AsyncGeneratorFunction
-    getPrototypeOf(async function* () {}),
+    ObjectGetPrototypeOf(async function* () {}),
     // TypedArray
-    getPrototypeOf(Uint8Array),
+    ObjectGetPrototypeOf(Uint8Array),
 
     // 18 The Global Object
     eval,
     isFinite,
+    // eslint-disable-next-line node-core/prefer-primordials
     isNaN,
-    parseFloat,
-    parseInt,
+    NumberParseFloat,
+    NumberParseInt,
```

Comment:
Those reference the global methods (same as `isNaN` and `isFinite`)
```suggestion
    // eslint-disable-next-line node-core/prefer-primordials
    parseFloat,
    // eslint-disable-next-line node-core/prefer-primordials
    parseInt,
```

Reply:
The thing is, <code>[Number.parseFloat] === [globalThis.parseFloat]</code> and <code>[Number.parseInt] === [globalThis.parseInt]</code>.

[globalThis.parseFloat]: https://tc39.es/ecma262/#sec-parsefloat-string
[globalThis.parseInt]: https://tc39.es/ecma262/#sec-parseint-string-radix
[Number.parseFloat]: https://tc39.es/ecma262/#sec-number.parsefloat
[Number.parseInt]: https://tc39.es/ecma262/#sec-number.parseint

Reply:
Interesting! It could be interesting to add a comment about that.